assets/src/scripts/components/map/index.js

```diff
@@ -140,6 +141,9 @@ const siteMap = function(node, {siteno, latitude, longitude, zoom}) {
     // Add a marker at the site location
     createMarker([latitude, longitude]).addTo(map);
 
+    //add nldi layers
+    addNldi(map, legendControl, siteno);
```

Comment:
I'd like to see a test to see that this layer gets created. See https://github.com/usgs/waterdataui/blob/master/assets/src/scripts/components/map/index.spec.js#L39, except it you won't look for a .leaflet-marker-pane. I'm not sure what gets created but you should be able to use developer tools to see.